root/usr/share/nethesis/NethServer/Language/en/NethServer_Module_SharedFolder.php

```diff
@@ -41,3 +41,4 @@
 $L['migrate-vhost_Message'] = 'This shared folder will be deleted and all data will be moved inside the virtual host. Migrate the shared folder to virtual host `${0}`?';
 $L['Migrate_label'] = 'Migrate';
 $L['__ANY__'] = 'default';
+$L['valid_platform,ibay-validator,ibay-name,3'] = 'The following characters are not allowed \ / : * ? " < > |';
```

Comment:
The label key must be fixed: ``valid_platform,ibay-create,ibay-name,3`` (``validator`` becomes ``create``).